pypika/dialects.py

```diff
@@ -104,7 +104,7 @@ def __copy__(self) -> "MySQLQueryBuilder":
 
     @builder
     def for_update(
-        self, nowait: bool = False, skip_locked: bool = False, of: TypedTuple[str, ...] = ()
+            self, nowait: bool = False, skip_locked: bool = False, of: TypedTuple[str, ...] = ()
```

Comment:
It seems like the pre-commit formatting hasn't been run. 

Locally, it can manually run with `pre-commit run --all`

@AzisK are you able to kick off the CI/CD?

Reply:
Thank you! Ran the pre-commit hook and pushed it up :)